apps/mobile/src/navigation/linking/index.js

```diff
@@ -1,6 +1,16 @@
 import { modalScreenName } from 'hooks/useIsModalScreen'
 import getStateFromPath from 'navigation/linking/getStateFromPath'
 import getInitialURL from 'navigation/linking/getInitialURL'
+import { isDev, isTest } from 'config'
+// TODO: Routing -- Tom -- This is becoming a special case. We have so far decided against constants
+// to keep the noise down, but it otherwise is a totally reasonable case for one. Let's just
+// keep doing it the way we have though (as plain strings) until we decide otherwise. 
+// The reality is that searching for 'Group Welcome' is mostly as reliable as searching for
+// GROUP_WELCOME_LANDING and has the same effect (hence why I've decided against constants for now).
+// Another alternative would be to define it either here or in the HomeNavigator as a const, as it
+// is the source of truth for the screen name, not data about the GroupWelcomeFlow...
+import { GROUP_WELCOME_LANDING } from 'screens/GroupWelcomeFlow/GroupWelcomeFlow.store'
```

Comment:
Sure thing. This was done over a year ago I think

Reply:
Oh no problem, it's a small thing just takes lots of words :) Appreciated. I am happy to make the update here, just wanted to make aware so not just tromping through